.flake8

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 100
```

Comment:
why 100? It was at 80 (which I believe is the standard) and I'm pretty sure it was at least mostly pep8 compliant.

Reply:
Most of the projects started using 100 for less hassle. Thus, I thought of same. Updated to defaults, agree that we can consider it separate request.